addons/stock_account/views/product_views.xml

```diff
@@ -38,15 +38,17 @@
                         <field name="property_cost_method" invisible="1"/>
                     </field>
                     <field name="standard_price" position="replace">
-                        <field name="standard_price"
-                            attrs="{'readonly':[('valuation','=','real_time')]}"
-                            widget='monetary' options="{'currency_field': 'currency_id', 'field_digits': True}"
-                            class="float-left"/>
-                        <div name="update_cost_price" class="col-lg-2">
-                            <button string="Update Cost" type="action"
-                            name="%(action_view_change_standard_price)d"
-                            attrs="{'invisible':['|', '|', ('valuation','!=', 'real_time'), ('cost_method', 'not in', ['average', 'standard']), ('product_variant_count', '>', 1)]}"
-                            class="oe_link oe_read_only"/>
+                        <div class="row  ml-0">
```

Comment:
why this change ? To align the button with the field ? Isn't it possible with the 'oe_inline' class ?